lndr-backend/src/Lndr/Types.hs

```diff
@@ -224,6 +225,10 @@ data ServerConfig = ServerConfig { lndrUcacAddr          :: !Address
                                  , awsPhotoBucket        :: !Text
                                  , awsAccessKeyID        :: !ByteString
                                  , awsSecrtAccessKey     :: !ByteString
```

Comment:
`awsSecrtAccessKey`… is this intentional or typo?

Reply:
lol, typo, thanks. using too much VIM autocomplete.